conanfile.py

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+from conans import ConanFile, CMake, tools
+from conans.errors import ConanInvalidConfiguration
+from conans.model.version import Version
+
+
+class AbseilConan(ConanFile):
+    name = "abseil"
```

Comment:
There was discussion on the associated issue about what to do with the "version" tag. I don't see it here--will folks using Abseil through Conan get an LTS, or a particular commit? Will they have to "sync" or "update" every time we push a new commit to GitHub?

Reply:
excellent question. This first commit is only related to provide Conan recipe itself, but don't build it using any CI service. As `version` is not listed in this recipe, you will need to specific the version tag when creating a new package, for example:

    conan create . abseil/<version>@abseil/stable

*version* could be your current date, commit, or even "latest". We would prefer using date e.g. *abseil/20181220@abseil/stable* because is easier to sort.

We could support LTS version too, but is outdated, when compared to your latest version, so you will be able to provide as many as you want.

I'll provide a new PR in the future, containing all changes needed to build 
Conan package in your CI, but as abseil-cpp doesn't have CI support yet, I'll need to wait for https://github.com/abseil/abseil-cpp/pull/197.

Reply:
Got it, thanks. We can hold off on deciding what to use for the version once we're ready to add it.